tests/common/aws_test_runner.c

```diff
@@ -210,9 +210,9 @@ static void RunTests( void )
     #endif
 
     #if ( testrunnerFULL_SERIALIZER_ENABLED == 1 )
```

Comment:
Should this macro be changed to `testrunnerSERIALIZER_UNIT_ENABLED` to give the right picture about these tests in the reports generate/analysis done from the CI test results?

Reply:
I don't think so. All of the other macros in this file are either of the form `testrunnerFULL...` or `testrunnerUTIL_PLATFORM...`, even when running unit tests